protocol-units/bridge/README.md

```diff
@@ -0,0 +1,120 @@
+# Current Contract Deployments
+## Holesky Ethereum Deployments
+
+| Contract                         | Address                                      |
+|-----------------------------------|----------------------------------------------|
+| `AtomicBridgeInitiatorMove.sol`   | 0xb33d9fA868054FD3F8F02ba9Fc04B2a1862Ab855   |
+| `AtomicBridgeCounterpartyMove.sol`    | 0xA6bB0d5eA03f3bEe3510EA3Ec39e85Cd7395Bc37   |
+
+
+## testnet.movementlabs Deployments
+the movement atomic-bridge and all of its modules are deployed under the same package address:
+`0xf7c46960ef7aa1b25cd5d48efef064525aa4ccec286fc3b3a7176bbfec09b84c`
+
+Verify this by running:
+```
+ movement account list --account 0xf7c46960ef7aa1b25cd5d48efef064525aa4ccec286fc3b3a7176bbfec09b84c 
+```
+## Running the Atomic Bridge Relayer In Production
+The atomic bridge relayer services requires a `config.json` at the root of the application inside a `./movement`
```

Comment:
The path to the bridge config file is `.movement/bridge`.  The config file in .movement is the suzuka node one. I made 2 separate config files because I didn't find how to mix both without mixing the Rust struct.